debian/patches/required-content_rating-and-releases-appdata.patch

```diff
@@ -0,0 +1,121 @@
+From 6f2d0cbe4342ad11333659e8b40cd56df2b586d7 Mon Sep 17 00:00:00 2001
+From: Ken VanDine <ken@vandine.org>
+Date: Tue, 7 Apr 2020 10:58:43 -0400
+Subject: [PATCH] Added required content_rating and releases to appdata, fixes
+ #153
+
+---
+ data/yelp.appdata.xml.in | 99 ++++++++++++++++++++++++++++++++++++++++
+ 1 file changed, 99 insertions(+)
+
+diff --git a/data/yelp.appdata.xml.in b/data/yelp.appdata.xml.in
+index 41742d27..24736a33 100644
+--- a/data/yelp.appdata.xml.in
++++ b/data/yelp.appdata.xml.in
+@@ -28,4 +28,103 @@
+   <compulsory_for_desktop>GNOME</compulsory_for_desktop>
+   <translation type="gettext">yelp</translation>
+   <developer_name>The GNOME Project</developer_name>
++  <content_rating type="oars-1.1" />
++  <releases>
++    <release version="3.36.0" date="2020-03-06" />
```

Comment:
Same comment as in https://github.com/endlessm/yelp/pull/65#issuecomment-693443336, but on hindsight we should probably do the same as debian upstream so don't bother - in any case debian/patches are ignored in our infra...

Reply:
:+1: